src/padding.css

```diff
@@ -0,0 +1,22 @@
+@import "./variables.css";
+
+.padding {
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.topDirection {
+  margin-top: var(--gutter);
+}
+
+.rightDirection {
+  margin-right: var(--gutter);
```

Comment:
yeah it's padding in the sense that affects the same area that would be considered padding in the box model.